encoding/preparation.md

```diff
@@ -162,8 +165,21 @@ VapourSynth-Script.
     extension for Jupyter Notebook.
 
     ![A Jupyter Notebook.](images/cnvimage101.png)
-
-3.  [AvsPmod](https://github.com/AvsPmod/AvsPmod/releases). This is the
+    
+3.  [VapourSynth Multi-Viewer](https://github.com/mysteryx93/VapourSynthViewer.NET/releases/).
+     Multi-Viewer is a very simple and elementary previewing tool. 
+     While the text editing is vastly inferior to VSEdit’s, 
+     and the preview lacks a lot of VSEdit’s advanced features,
+     its tab-based previewing functionality easily outclasses VSEdit’s single preview window, 
+     because it makes comparisons between different script versions a lot more convenient and efficient.
+     
+     In short: very useful for thorough comparisons and filter fine-tuning, not so much for everything else.
+     
+     ![The editing window](images/VS-Multi-Viewer_Preview.png)
+     ![the preview window](images/VS-Multi-Viewer_Editor.png)
```

Comment:
The title text is used for the figure caption, so it should start with an upper-case letter. As such, mentioning the application's name would also be useful for clarification.

Images need to be surrounded by blank lines.